cubequery/__init__.py

```diff
@@ -30,6 +35,15 @@ def get_config(section, key):
 
     return _config.get(section, key)
 
+def fetch_form_settings():
+    """
+    Gets dynamic form settings loaded from a JSON file
+    """
+    global settings_json
+    if settings_json == "":

```

Comment:
Don't need this. The file is opens on file init. settings_json will have already been loaded.